From 3f3e9a546fa5e9a335ad3dc9afc805fd4d2809fd Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date: Sun, 12 May 2019 08:40:53 +0200
Subject: [PATCH] Fix build with exiv2-0.27.1

Summary: Upstream is moving things around between headers. While they are busy doing that, they recommend including exiv2.hpp.

Reviewers: #gwenview

Tags: #gwenview

Differential Revision: https://phabricator.kde.org/D21148
---
 lib/imagemetainfomodel.cpp | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/lib/imagemetainfomodel.cpp b/lib/imagemetainfomodel.cpp
index d7b89ab2..58f1c13d 100644
--- a/lib/imagemetainfomodel.cpp
+++ b/lib/imagemetainfomodel.cpp
@@ -33,9 +33,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 #include <KFormat>
 
 // Exiv2
-#include <exiv2/exif.hpp>
-#include <exiv2/image.hpp>
-#include <exiv2/iptc.hpp>
+#include <exiv2/exiv2.hpp>
 
 // Local
 #ifdef HAVE_FITS
-- 
2.21.0

